internal/log_analysis/athenaviews/athenaviews.go

```
@@ -169,7 +170,7 @@ func newPantherViewColumns(tables []*awsglue.GlueTableMetadata, extraColumns []a
 }
 func (pvc *pantherViewColumns) inferViewColumns(table *awsglue.GlueTableMetadata, extraColumns []awsglue.Column) {
 	// NOTE: in the future when we tag columns for views, the mapping  would be resolved here
-	columns, _ := awsglue.InferJSONColumns(table.EventStruct(), awsglue.GlueMappings...)
+	columns, _ := glueschema.InferColumns(table.EventStruct())
```

We shouldn't ignore the error here. This points to a problem.

---

The main problem is that this code does not return an error in the first place. I can only panic, or refactor this as well.
The error won't be a problem here since we have already checked the log types on registration/creation.

---

The `glueschema.InferColumns` returns an error . Even if it now panics instead of returning one, this behavior might be altered in the future. 
The `InferColumns` will change overtime and it will rely on the callers to handle errors appropriately.

---

refactored to return errors